package-testing/sdk-test-runner/test-sdk.sh

```diff
@@ -139,6 +155,7 @@ case "$command" in
         echo "  ... Starting Test Cluster node [Eppo-exp/sdk-test-runner]"
         
         docker run \
+          --add-host host.docker.internal:host-gateway \
```

Comment:
docker on github actions is a little bit different. This is one of those gotcha differences